### ISSUE TITLE
Usage of bail() in validation schema

### ISSUE THREAD

**Issue by hamboomger** [express-validator/express-validator#848]
Hello there!
Is there a way to get similar behaviour to what bait() method does using Schema Validation? The desired behaviour is to stop validation chain after a specific validation failed (`exists` in my case)

**Comment by ngocketit** [comment]
Like this:
```
  genres: {
    exists: {
      errorMessage: 'is required',
    },
    bail: true,
    isArray: true,
    custom: {
      options: (value: any[]) => {
        if (value.length <=0) return true
        return value.every(item => typeof item === 'string')
      }
    },
    errorMessage: 'must be array of string',
  },
```
However, there is no way to use multiple `bail` in schema. I think one solution is to have `bail` in `options`

**Comment by ngocketit** [comment]
This seems to work (e.g, error stops after `exists`) but if used with TypeScript, it won't be a valid type.

**Comment by hamboomger** [comment]
@ngocketit Yea, I actually use typescript in this project. What about making it a field inside the validation type, like this:
`
genres: {
    exists: {
      errorMessage: 'is required',
      bail: true,
    },
    isArray: {
      bail: true
    },
    custom: {
      options: (value: any[]) => {
        if (value.length <=0) return true
        return value.every(item => typeof item === 'string')
      }
    },
    errorMessage: 'must be array of string',
  },
`
I can try to implement this feature if you don't mind

**Comment by ngocketit** [comment]
@hamboomger That's very nice & it'd be awesome if it can be implemented.

**Comment by tommmn** [comment]
I already have implemented it.
I will PR soon

**Comment by hamboomger** [comment]
@tommmn cool, thanks for mentioning it here!

**Comment by tommmn** [comment]
@hamboomger I've open PR. should wait for a merge if there are no problems.

**Comment by hamboomger** [comment]
Oh you actually referenced this issue in your PR, sorry, didn't notice

**Comment by gustavohenke** [comment]
Published in v6.9.0!
